### ISSUE TITLE
Show used vscode version on code-server startup

### ISSUE THREAD

**Issue by frank-dspeed** [cdr/code-server#991]
I Would like to implament something to show the used vscode version for building on code-server startup

**Comment by code-asher** [comment]
One idea would be to just run --version before starting code server. So if this was bash you could do something like:

```bash
version=$(code-server --version --format json | jq .vscodeVersion)
echo "VS Code version is $version"
code-server
```

**Comment by frank-dspeed** [comment]
so we should modify the CMD of the current image to code-server --version && code-server ? because the intent behind this is i just tested the one liner and was not sure if i will get v2 with latest.

I Think its essential to see on startup what version the docker image is running .

**Comment by elgandoz** [comment]
Essential?

**Comment by sr229** [comment]
@frank-dspeed Docker best practices is to use versioned tags. Our tags reflect our Docker deploys, hence we don't really need this implemented if you *know* what tag you used for deploying.

**Comment by sr229** [comment]
Closing this as `wontfix`. Just adhere to Docker best practices. If you don't like to do versioning, consider using `latest` instead.

**Comment by code-asher** [comment]
@frank-dspeed yeah if you're just trying to find out what version the `v2` or `latest` tags are running you could run the one-liner with the version flag to verify.

```
docker run --rm -it codercom/code-server:v2 --version
```

The VS Code version also appears in the `Help > About` menu in the IDE. Hope that helps. :hamburger:

**Comment by frank-dspeed** [comment]
so you all vote for not showing that on startup ? Ok as i now know what to do its ok for me thanks a lot.
